bin/main

```diff
@@ -1,2 +1,61 @@
 #!/usr/bin/env ruby
-puts 'Hello world!'
+require 'colorize'
+class GameUi
+  attr_reader :title, :instructions
+
+  def initialize(title, instructions)
+    @title = title
+    @instructions = instructions
+  end
+
+  def display_title_on_screen
+    puts title
+  end
+
+  def provide_players_info
+    puts 'Please provide the name for Player1: '
+    player1_name = gets.chomp
+    puts 'Please choose between X or O '
+    player1_value = gets.chomp
+    puts 'Please provide the name for Player2: '
+    player2_name = gets.chomp
+    player2_value = player1_value == 'O' ? 'X' : 'O'
+    infoplayer = {
+      player1_name => player1_value,
+      player2_name => player2_value
+    }
+    display_info(infoplayer)
+  end
+
+  def display_instructions
+    puts instructions
+  end
+
+  def display_gameboard
+    puts ' 1|2|3'.yellow
+    puts ' ----- '.yellow
+    puts ' 4|5|6'.yellow
+    puts ' ----- '.yellow
+    puts ' 7|8|9'.yellow
+  end
```

Comment:
You also have an error here due to the colorize gem, if you remove the gem remember to delete the colours in those lines.